nodewatcher/modules/monitor/http/survey/management/commands/export_survey_data.py

```diff
@@ -0,0 +1,81 @@
+import io
+import json
+import time
+
+from django.core.management import base
+
+from django_datastream import datastream
+
+
+class Command(base.BaseCommand):
+    help = 'Consolidates survey data from all nodes and exports the meta graph as a JSON file.'
+
+    def add_arguments(self, parser):
+
+        # Optional arguments
+        parser.add_argument(
+            '--surveys-to-export',
+            type=int,
+            action='store',
+            dest='number',
+            default=1,
+            help='Number of surveys to export'
+        )
+
+        parser.add_argument(
+            '--label',
+            type=str,
+            action='store',
+            dest='label',
+            help='Optional label for the filename',
+        )
+
+    def handle(self, *args, **options):
+        """
+        Exports the latest survey data graph as a JSON file into the root directory.
+        """
+
+        number = options['number']
+        label = options['label']
+
+        streams = datastream.find_streams({'module': 'monitor.http.survey'})
+
+        # Create a meta graph
+        meta_vertices, meta_edges = [], []
+        friendly_nodes = []
+        for stream in streams:
+            datapoint_iterator = datastream.get_data(
```

Comment:
You could have a bash for loop which goes and creates timestamps for every two hours and gives them to this script? Or you would really like to search for latest n different surveys.

If you need that, then feel free to do it. Then make timestamp + how many different survey graphs you want from that timestamp into the past.